pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -1000,6 +1004,20 @@ func (k *Bootstrapper) UpdateNode(cfg config.ClusterConfig, n config.Node, r cru
 	return nil
 }
 
+func (k *Bootstrapper) copyResolvConf(cfg config.ClusterConfig) error {
```

Comment:
does this happen for both Kic and VM dirvers?

Reply:
lets add a comment and say why this function exsists

Reply:
It does, I was thinking that incase there was a way the KIC driver ends up in the same state we can prevent the issue from occuring.

Reply:
> lets add a comment and say why this function exsists

Added the following comment:
```
// copyResolvConf is a workaround for a regression introduced with https://github.com/kubernetes/kubernetes/pull/109441
// The regression is resolved by making a copy of /etc/resolv.conf, removing the line "search ." from the copy, and setting kubelet to use the copy
// Only Kubernetes v1.25.0 is affected by this regression
```